code/modules/food_and_drinks/drinks/drinks.dm

```diff
@@ -513,6 +513,11 @@
 	volume = 20
 	amount_per_transfer_from_this = 5
 	isGlass = FALSE
+	/** 
+	  * The var fixes the lean sprite to work properly upon crafting. Before crafting, it will 
+	  * display the random sprite; however after crafting it the random sprite becomes false 
+	  * and displays the lean sprite
+	  */ 
```

Comment:
bruh https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md#documenting-a-variable

Reply:
> 
> 
> bruh https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md#documenting-a-variable

there you I commented it properly and made it 100% direct to the person who reads it